packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -64,6 +64,7 @@ class _FutureMemoryImage extends ImageProvider<_FutureMemoryImage> {
 
   /// See [ImageProvider.load].
   @override
+  // ignore:deprecated_member_use
```

Comment:
Could you put a TODO about replacing this once the new API is available on stable?

Reply:
Yes, I was creating the issue, let me push a quick fix.